$MirOS: ports/devel/subversion/patches/patch-subversion_libsvn_repos_repos_c,v 1.2 2007/09/17 21:02:06 bsiegert Exp $

	Help sucky operating systems to be less sucky.

	In general, if the command
	$ env -i LC_CTYPE=en_US.UTF-8 /usr/bin/locale >&-
	fails - you can check this for example with
	$ env -i LC_CTYPE=nonexistant /usr/bin/locale >&-
	then you need to take some action, e.g.
	# localedef -c -i en_US -f UTF-8 en_US.UTF-8
	(or with sudo), if that works.

--- subversion/libsvn_repos/repos.c.orig	Fri Apr  4 18:04:56 2008
+++ subversion/libsvn_repos/repos.c	Fri Jun 27 23:02:41 2008
@@ -341,9 +341,15 @@ create_hooks(svn_repos_t *repos, apr_poo
 "# "                                                                         NL
 HOOKS_ENVIRONMENT_TEXT
 "# "                                                                         NL
+"# Its standard output and standard error are not displayed"                 NL
+"# to the client either."                                                    NL
+"# "                                                                         NL
 "# Here is an example hook script, for a Unix /bin/sh interpreter."          NL
 PREWRITTEN_HOOKS_TEXT
 ""                                                                           NL
+""                                                                           NL
+"# If this breaks, consider the documentation on how to create this locale." NL
+"export LC_CTYPE=en_US.UTF-8"
 ""                                                                           NL
 "REPOS=\"$1\""                                                               NL
 "USER=\"$2\""                                                                NL
